client/app/components/LogIn.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { Input, InputField, FormControl, VStack, Text, InputSlot, InputIcon, Button, ButtonText } from "@gluestack-ui/themed";
+import { EyeIcon, EyeOffIcon } from 'lucide-react-native';
+
+interface LogInProps {
+  onPress: () => void;
+}
+
+export const LogIn: React.FC<LogInProps> = ({ onPress }) => {
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+  const [signUp, setSignUp] = useState<boolean>(false);
+
+  const handleState = () => {
+    setShowPassword((showState) => {
+      return !showState;
+    });
+  };
+
+  const handleType = () => {
+    setSignUp((typeState) => {
+      return !typeState;
+    });
+  }
+
+  return (
+    <FormControl p='$4' width="80%">
+      <VStack space='xl'>
+        <VStack space='xs'>
+          <Text color='$textLight200' lineHeight='$xs'>
+            Email
+          </Text>
+          <Input>
+            <InputField
```

Comment:
Currently, when I type something in the input field, the font color is very dark. Is there a way to change this to lighter colors?